.github/workflows/build-py-dists.yml

```diff
@@ -8,7 +8,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [windows-latest, macos-latest]
-                python-version: [3.7, 3.8, 3.9]
+                python-version: [3.7, 3.8, 3.9, "3.10"]

```

Comment:
Curious why the quotes are needed? And do the other versions need them as well?

Reply:
Oh man, this one was hilarious to watch play out.  The first time I tried to add `3.10` to the build matrix, I was _shocked_ to see that GHA had added not Python 3.10, but Python 3.1 and everything failed.  It turns out that the YML parser tries to do some intelligent casting, so the quotes are needed here to specify that we do indeed want the string `"3.10"` and not the float `3.1`.